### DISCUSSION TITLE
Internal method for fetching defined route params?

### DISCUSSION THREAD

**Discussion by kennonb** [laravel/nova-issues#4855] [Q&A] [answered]
Hi,

I'm trying to convert a custom tool from Nova 3.x to 4.x. The tool was using the following param while in Nova 3.x: `Nova.app.route.params.id`. This was pulling the {id} param from a defined route in the Vue.js router. Now that Nova 4.x is using Inertia.js for the routing, is there a similar way to grab this param? Or should I revert to parsing the the URL?

Thanks!

**Comment by crynobone** [comment, marked as answer]
You can pass the params to Vue from any Inertia response and it would be available as props. You should see this being done under `Laravel\Nova\Http\Controller\Pages\*Controller` and `resources/js/pages/*.vue`
